recipes/ngscheckmate/meta.yaml

```diff
@@ -1,33 +1,40 @@
 package:
   name: ngscheckmate
-  version: 1.0.0
+  version: 1.0.1
 
 source:
-  url: https://github.com/parklab/NGSCheckMate/archive/8ea2c043896adf9bbd1dc5bb2136f3a1dab9b445.zip
-  md5: 4c4e34abc8c4851d29661f5f0cc13b49
+  url: https://github.com/parklab/NGSCheckMate/archive/51e2634ec50d659980dcb2f3274046d4219fabc7.zip 
+  md5: a2caabb0fa5f4db43af8700b17ff1b7f
 
 build:
-  number: 4
+  number: 0
   noarch: generic
 
 requirements:
   host:
-    - python 2.7.18
-    - R 4.1
-    - samtools 1.14
+    - python 2.*
```

Comment:
Python 2 is really out of date and is not supported anymore. Is this package still maintained?

Reply:
I can't speak for the developers but the package does not seem to be regularly updated. It happens to be in python 2 which is old, but the package is still quite useful.

Reply:
Yes, the package is really useful, but unfortunately is basically abandonware. I'm using it currently in Nextflow pipelines, and I'm after an updated container.